Tests/LibGC/TestGCBindings.swift

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2024, Andrew Kaster <andrew@ladybird.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+import AK
+import GC
+import GCTesting
+import Testing
+
+// FIXME: We want a type declared *here* for HeapString, but it gives a compiler warning:
+//  error: type 'GCString' cannot conform to protocol 'HeapAllocatable' because it has requirements that cannot be satisfied
+//  Even using the same exact code from LibGC/Heap+Swift.swift
```

Comment:
I think this means that one of the C++ types on which the HeapAllocatable protocol depends isn't getting imported.